src/DynamoDbModel.php

```diff
@@ -155,11 +155,57 @@ public function save(array $options = [])
         return $saved;
     }
 
+    /**
+     * Saves the model to DynamoDb asynchronously
+     * And returns a promise
+     * @param array $options
+     * @return bool|\GuzzleHttp\Promise\Promise
```

Comment:
Remove `bool`?

Reply:
Because it can return `false` at line 169, 173, 177